RedHatInsights/patchman-ui
# chore(deps): Update frontend-components

# Description

Associated Jira ticket: [SPM-2051](https://issues.redhat.com/browse/SPM-2051)

Updated `Frontend Components` to fix an UI Issue.
With the package update came deprecation of the `<Main>` component so I had to replace it with `section` with proper styling.

# How to test the PR
1. Make sure all filters in all pages have correct styling and don't look like this (mainly the `Tags` and `Operation System` filters):
![image](https://github.com/RedHatInsights/patchman-ui/assets/20592948/678b0ede-b571-4ef5-8200-974c8cb4b3d1)

2. Verify that tables etc. have correct padding in all pages (red)
![image](https://github.com/RedHatInsights/patchman-ui/assets/20592948/98c9c62e-a762-433c-80cc-21f1f644af57)


# Before the change


# After the change


# Dependent work link


# Checklist:
